[BEGIN] 2018/12/2 15:30:19
[root@centos7evcloud ~]# python36  test_token2.py
Sun Dec  2 15:30:23 2018
共计上传 2000000 次 成功次数 1 上传成功时间 Sun Dec  2 15:30:23 2018
共计上传 2000100 次 成功次数 101 上传成功时间 Sun Dec  2 15:30:48 2018
共计上传 2000200 次 成功次数 201 上传成功时间 Sun Dec  2 15:31:16 2018
Traceback (most recent call last):
  File "test_token2.py", line 20, in <module>
    new1=json.loads(new)
  File "/usr/lib64/python3.6/json/__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python3.6/json/decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python3.6/json/decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

[END] 2018/12/2 15:34:53
